l10n/en-US/viewer.ftl

```diff
@@ -702,3 +682,87 @@ pdfjs-editor-edit-comment-dialog-cancel-button = Cancel
 
 pdfjs-editor-add-comment-button =
     .title = Add comment
+
+## The view manager is a sidebar displaying different views:
+##  - thumbnails;
+##  - outline;
+##  - attachments;
+##  - layers.
+## The thumbnails view is used to edit the pdf: remove/insert pages, ...
+
+pdfjs-toggle-views-manager-button =
+    .title = Toggle Sidebar
+pdfjs-toggle-views-manager-notification-button =
+    .title = Toggle Sidebar (document contains outline/attachments/layers)
+pdfjs-toggle-views-manager-button-label = Toggle Sidebar
+
+pdfjs-views-manager-sidebar =
+    .aria-label = Sidebar
+pdfjs-views-manager-view-selector-button =
+    .title = Views
+pdfjs-views-manager-view-selector-button-label = Views
+pdfjs-views-manager-pages-title = Pages
+pdfjs-views-manager-outlines-title = Document Outline
+pdfjs-views-manager-attachments-title = Attachments
+pdfjs-views-manager-layers-title = Layers
+
+pdfjs-views-manager-pages-option-label = Pages
+pdfjs-views-manager-outlines-option-label = Document outline
+pdfjs-views-manager-attachments-option-label = Attachments
+pdfjs-views-manager-layers-option-label = Layers
+
+pdfjs-views-manager-add-file-button =
+    .title = Add file
+pdfjs-views-manager-add-file-button-label = Add file
+
+# Variables:
+#   $count (Number) - the number of selected pages.
+pdfjs-views-manager-status-action-label =
+    { $count ->
+        [one] { $count } selected
+        *[other] { $count } selected
+    }
+pdfjs-views-manager-status-none-action-label = Select pages
+pdfjs-views-manager-status-action-button-label = Manage
+pdfjs-views-manager-status-copy-button-label = Copy
+pdfjs-views-manager-status-cut-button-label = Cut
+pdfjs-views-manager-status-delete-button-label = Delete
+pdfjs-views-manager-status-save-as-button-label = Save as…
+
+# Variables:
+#   $count (Number) - the number of selected pages to be cut.
+pdfjs-views-manager-status-undo-cut-label =
+    { $count ->
+        [one] 1 page cut
+        *[other] { $count } pages cut
+    }
+
+# Variables:
+#   $count (Number) - the number of selected pages to be copied.
+pdfjs-views-manager-status-undo-copy-label =
+    { $count ->
+        [one] 1 page copied
+        *[other] { $count } pages copied
+    }
+
+# Variables:
+#   $count (Number) - the number of selected pages to be deleted.
+pdfjs-views-manager-status-undo-delete-label =
+    { $count ->
+        [one] 1 page deleted
+        *[other] { $count } pages deleted
+    }
+
+pdfjs-views-manager-status-waiting-ready-label = Getting your file ready…
+pdfjs-views-manager-status-waiting-uploading-label = Uploading file…
+pdfjs-views-manager-current-outline-item-button =
+    .title = Find Current Outline Item
```

Comment:
What is an outline item?

Reply:
It's an entry in the document outline.
In most of the documents I saw the outline is almost a table of contents but according to the pdf terminology an outline item is `sometimes called bookmarks` (it's from the specs). In Acrobat Reader the panel dedicated to document outline is labelled "Bookmarks".
Anyway this string was already existing and I just changed its name (see https://searchfox.org/firefox-main/source/toolkit/locales/en-US/toolkit/pdfviewer/viewer.ftl#201).
That said when I see the current translation in french (which is really bad, tbh I don't understand what it means), we need to have some help from the content team about that.